app/src/main/java/com/example/goodnote/utils/Functions.kt

```diff
@@ -1,2 +1,8 @@
 package com.example.goodnote.utils
 
+import com.example.goodnote.database.models.*
+
+internal fun List<Tag>.toTagsString(): String {
+    val names: List<String> = this.map { it.name }
```

Comment:
It might be more Kotlin code style if you inline this function. 
Example:
```
internal fun List<Tag>.toTagsString(): String = map { it.name }.joinToString(separator = ", ")
```

If you want to go crazy, there is even shorter notation, since there is existing `joinToString` function which accepts lambda :) 
Example:
`internal fun List<Tag>.toTagsString(): String = joinToString(separator = ", ") { it.name }`